website/src/components/SequenceDetailsPage/SequencesDataTableTitle.astro

```diff
@@ -29,16 +29,33 @@ const { sequenceEntryHistory, accessionVersion, showFastaDownload } = Astro.prop
             )
         }
         {
-            showFastaDownload && (
-                <a
-                    class='sm:outlineButton inline-block'
-                    href={routes.sequencesFastaPage(accessionVersion) + '?download'}
-                >
-                    <span class='hidden sm:inline'>Download FASTA</span>
-                    <span class='sm:hidden inline text-xl'>
+            showDownload && (
```

Comment:
Also maybe it would be nice to just reuse the react DownloadButton component here, but they handle responsiveness differently and are also styled slightly differently and for now I couldn't be bothered to "merge" the two buttons into a single component. If desired I can put some more work into this refactoring so we end up with a single component.